rollup.config.js

```diff
@@ -200,22 +197,33 @@ function buildTarget(buildType, moduleFormat) {
         name: 'pc'
     };
 
+    const sdkVersion = {
+        _CURRENT_SDK_VERSION: version,
+        _CURRENT_SDK_REVISION: revision
+    };
+
     const jsccOptions = {
         debug: {
             values: {
-                _DEBUG: 1,
-                _PROFILER: 1
+                ...sdkVersion,
+                ...{
+                    _DEBUG: 1,
+                    _PROFILER: 1
+                }
             },
-            keepLines: true
+            keepLines: true,
+            sourcemap: 'inline'
```

Comment:
`sourcemap` is a `Boolean` and not a `string` and by default `true`, so this can be completely removed:

https://github.com/aMarCruz/jscc/blob/7f8634b9677097c0e16311caae27937ff69d7f51/index.d.ts#L86-L93